docs/docs-next/docs/concepts/assets/asset-checks.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Asset checks"
+sidebar_position: 7
```

Comment:
I usually do these in increments of 10. That way when you decide you want to add something between 7 and 8, you don't have to go and change everything above and below but you just add one called 71.